File: iOS Examples/Objective-C/EffectsProcessorDemo/EffectsProcessorDemo/Effects View Controllers/MoogLadderViewController.m
//
//  MoogLadderViewController.m
//  EffectsProcessorDemo
//
//  Created by Aurelius Prochazka on 1/29/15.
//  Copyright (c) 2015 Aurelius Prochazka. All rights reserved.
//

#import "MoogLadderViewController.h"
#import "SharedStore.h"

@implementation MoogLadderViewController {
    IBOutlet AKPropertySlider *cutoffFrequencySlider;
    IBOutlet AKPropertySlider *resonanceSlider;
    IBOutlet AKPropertySlider *mixSlider;
}

- (void)viewDidLoad
{
    SharedStore *global = [SharedStore globals];

    cutoffFrequencySlider.property = global.moogLadder.cutoffFrequency;
    resonanceSlider.property       = global.moogLadder.resonance;
    mixSlider.property             = global.moogLadder.mix;
}



@end

File: iOS Examples/Swift/EffectsProcessorDemo/EffectsProcessorDemo/MoogLadderViewController.swift
//
//  MoogLadderViewController.swift
//  EffectsProcessorDemo
//
//  Created by Matthew Fecher on 5/23/15.
//  Copyright (c) 2015 audiokit.io. All rights reserved.
//

import UIKit

class MoogLadderViewController: UIViewController {
    
    @IBOutlet weak var cutoffFrequncySlider: AKPropertySlider!
    @IBOutlet weak var resonanceSlider: AKPropertySlider!
    @IBOutlet weak var mixSlider: AKPropertySlider!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        let global = SharedStore.globals
        
        cutoffFrequncySlider.property = global.moogLadder!.cutoffFrequency
        resonanceSlider.property = global.moogLadder!.resonance;
        mixSlider.property = global.moogLadder!.mix;
        
    }
    
    
}
